### ISSUE TITLE
MassEnrollmentWorker throws Sidekiq warning

### ISSUE THREAD

**Issue by ragesoss** [WikiEducationFoundation/WikiEduDashboard#4781]
### What is happening?
The tests for MassEnrollmentController trigger a Sidekiq warning for MassEnrollmentWorker:

```
WARN: Job arguments to MassEnrollmentWorker do not serialize to JSON safely. This will raise an error in
Sidekiq 7.0. See https://github.com/mperham/sidekiq/wiki/Best-Practices or raise an error today
by calling `Sidekiq.strict_args!` during Sidekiq initialization.
```

### To Reproduce

Run the rspec tests for MassEnrollmentController.

**Comment by aaditya9899** [comment]
Hey @chepyego let me know if the issue can't be reolved ,thankx.

**Comment by aaditya9899** [comment]
hey is someone working on it still?

**Comment by chepyego** [comment]
hey yes am working on it, sorry  i was having some troubles setting up the
application

**Comment by marklocklear** [comment]
@chepyego I think you just need to change the text in app/services/add_users.rb and b/app/services/join_course.rb from symbols to regular strings; so instead of ` { failure: 'Not an existing user.' }` use `{ "failure" => 'Not an existing user.' }`

**Comment by chepyego** [comment]
@marklocklear am actually not getting any sidekiq warning and i haven't changed anything
![massEnrollement](https://user-images.githubusercontent.com/22706319/156446833-c0cd341b-5491-48e0-b2db-9ebb9dbcf6bd.png)

**Comment by ragesoss** [comment]
It looks like you ran the spec for SelfEnrollmentController, not MassEnrollmentController.

**Comment by ragesoss** [comment]
Fixed in 1c44a30d4c5b3c3a0ce6f26e68be2ddc77516dc0
